app/components/Review/Accordion.tsx

```diff
@@ -112,7 +112,7 @@ const Accordion = ({
   // ugly and I don't like it
   useEffect(() => {
     setIsToggled(getToggledState(toggled, defaultToggled));
-  }, [toggled]);
+  }, [toggled, defaultToggled]);
```

Comment:
I, personally, find it weird to add to dependencies something that does not get updated in the component but only through props (meaning component re-render anyway) ... oh, those Lint rules